lib/absinthe/phoenix/channel.ex

```diff
@@ -34,7 +34,10 @@ defmodule Absinthe.Phoenix.Channel do
     absinthe_config =
       Map.put(absinthe_config, :pipeline, pipeline || {__MODULE__, :default_pipeline})
 
-    socket = socket |> assign(:absinthe, absinthe_config)
+    socket =
```

Comment:
Not sure this change is necessary but ok. :)

Reply:
Huh. I probably changed it at some point then changed it back with different formatting. Thanks.